ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1097,6 +1097,21 @@ void GCS_MAVLINK_Copter::handle_manual_control_axes(const mavlink_manual_control
     manual_override(copter.channel_yaw, packet.r, 1000, 2000, tnow);
 }
 
+// this method is to guard against completely ridiculous values being
```

Comment:
I think we should change this comment to something like, "sanity check velocity or acceleration vector components are all number (e.g. not NaN) and below 1000.  vec argument units are in meters."

Reply:
Done - but I've used the correct units